tests/e2e/device/FreeRTOSConfig.h

```diff
@@ -0,0 +1,204 @@
+/*

```

Comment:
Nit: Could we put FreeRTOS Config type stuff into like a tests/common?  I'm worried that if we ever add new test cases, we'll end up with bunches of copies of this floating around when we really we should just need one?

If this ends up making stuff gross or we want to see till we get >1 test scenario that needs it, I'm fine if you punt this.

Reply:
ack